main/org.thingpedia.iot.climate/manifest.tt

```diff
@@ -1,16 +1,18 @@
-abstract class @org.thingpedia.iot.thermostat
-#_[thingpedia_name="Thermostat"]
-#_[thingpedia_description="Interface for Thermostat."]
-#_[description="Interface for Thermostat."]
+abstract class @org.thingpedia.iot.climate
+#_[thingpedia_name="cClimate"]
```

Comment:
Typo?
Also, I like the older name, it's more descriptive for users...

Reply:
climate is how home assistant name it. yes typo, I'll fix it